microsoft-edge/devtools-guide-chromium/whats-new/2023/02/devtools-110.md

```diff
@@ -0,0 +1,175 @@
+
+![Taking a heap snapshot in the Memory tool](./devtools-110-images/faster-heap-snapshot.png)
+
+See also:
+* [Record heap snapshots using the Memory tool](../../../memory-problems/heap-snapshots.md#take-a-snapshot)
+
+
+<!-- ====================================================================== -->
+## Move tools between the Activity Bar and Quick View more easily in Focus Mode
+
+<!-- Subtitle: Customize both the Activity Bar and Quick View in Focus Mode to see only the tools you care about. -->
+
+Microsoft Edge 110 contains various improvements to **Focus Mode** in DevTools.  The following changes make it easier to customize the Activity Bar and Quick View, to show only the tools that you're currently interested in.
+
+
+<!-- ------------------------------ -->
+#### Right-click menu shows the current location of the other toolbar to move a tool to
+
+The right-click menu for moving an optional tool between Quick View and the Activity Bar now includes the position of the Activity Bar, such as:
+* **Move to bottom Quick View**
+* **Move to side Quick View**
+* **Move to left Activity Bar**
+* **Move to top Activity Bar**
+
+![Right-click menu showing 'Move to bottom Quick View'](./devtools-110-images/move-to-bottom-quick-view.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ------------------------------ -->
+#### Tool name remains visible when you right-click the tool icon
+
+The context menu for moving a tool between the Activity Bar and Quick View no longer obscures the tool's title in the toolbar:
+
+For example, the **Network Conditions** tool name remains visible when you right-click the tool's tab in Quick View:
+
+![The right-click menu in Quick View](./devtools-110-images/focus-mode-moving-tools-context-menu.png)
+
+As another example, the **Network** tool name remains visible when you right-click the tool's tab in the Activity Bar:
+
+![The right-click menu in the Activity Bar](./devtools-110-images/tool-name-remains-visible.png)
+
+See also:
+* [Simplify DevTools using Focus Mode](../../../experimental-features/focus-mode.md)
+
+
+<!-- ------------------------------ -->
+#### Focus is preserved in Activity Bar or Quick View when moving a tool between them
+
+For keyboard shortcut and assistive technology users, focus is preserved in the Activity Bar or Quick View when moving the currently selected tool from one to the other or removing it.
+<!-- todo: is focus preserved within the tool, or within the toolbar? -->
+
+<!-- todo: screenshot?
+![alt text](./devtools-110-images/filename.png)
+-->
+
+See also:
```

Comment:
See my previous comment about the third See also link. I don't think we need this content 3 times.